src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -94,7 +94,8 @@ proc createFetchMoreMessagesItem(self: Module): Item =
     @[],
     newTransactionParametersItem("","","","","","",-1,""),
     @[],
-    TrustStatus.Unknown
+    TrustStatus.Unknown,
+    false
```

Comment:
Could you add the name of the property for clarity. In this case, it would be `ensVerified = false` if I'm not mistaken